libraries/dawg-house/src/main/java/com/comcast/video/dawg/controller/house/HouseRestController.java

```diff
@@ -114,7 +114,9 @@
 
     public static final String[] ignoreProps = new String[] { "--class",
         PersistableDevice.RESERVER, PersistableDevice.EXPIRATION_KEY,
-        MetaStb.CATSSERVERHOST,MetaStb.MACADDRESS, MetaStb.ID, MetaStb.MODIFIED_PROPS};
+        MetaStb.CATSSERVERHOST,MetaStb.MACADDRESS, MetaStb.ID, MetaStb.MODIFIED_PROPS,
+        MetaStb.CATSKEYSETMAPPING
```

Comment:
because i figured, it's a setting that should not be messed with without knowledge what it does.